### DISCUSSION TITLE
Where a keyboard navigation `F6` for useLandmark comes from?

### DISCUSSION THREAD

**Discussion by takuyahara** [adobe/react-spectrum#6686] [Q&A] [answered]
Until reading [useLandmark](https://react-spectrum.adobe.com/react-aria/useLandmark.html) I had no idea about landmark at all and found it convenient, but am curious where a keyboard navigation `F6` comes from. AFAIK keybinds are not defined in [ARIA APG](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/) and hitting `F6` in [ARIA Landmarks Example](https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/search.html) results nothing (I use Brave in macOS).

**Comment by reidbarber** [comment, marked as answer]
This is from @majornista:

> I think a principle reason for using F6/Shift+F6 to move between landmarks within Web pages is that it was already used to move between panels within desktop applications, like Acrobat and MS Office apps, and frames within browsers. The WAI-ARIA APG even suggests F6 as a way for moving between panes of a window splitter: https://www.w3.org/WAI/ARIA/apg/patterns/windowsplitter/#keyboard-interaction-26.

**Reply by takuyahara** [reply]
@reidbarber Thank you for your super prompt answer. Can I see the original discussion of the quote?

**Reply by reidbarber** [reply]
This was from a previous discussion that came up internally. I can bring the case you mentioned up with the team to see if it would be a good idea to look into an additional key combination so that this covers all browsers and operating systems.

**Reply by takuyahara** [reply]
@reidbarber All clear, thank you so much 🙏. Then what do you mean "the case you mentioned up"?

**Reply by reidbarber** [reply]
I was referring to Brave MacOS not working, but now I realize you were referring to the APG example and not useLandmark, so you can ignore that.
